packaging/install_dependencies.sh

```diff
@@ -2,12 +2,22 @@
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ROS_VERSION=$(rosversion -d)
-if [ "$ROS_VERSION" = "noetic" ]; then
+if [ "$ROS_VERSION" = "noetic" -o "$ROS_VERSION" = "foxy" ]; then
     PYTHON_SUFFIX=3
 else
     PYTHON_SUFFIX=""
 fi
 
+if [ "$ROS_VERSION" = "foxy" ]; then
```

Comment:
Here I would also use not equal to melodic or noetic to be more robust against changes